### ISSUE TITLE
could some folding keywords be added?

### ISSUE THREAD

**Issue by RichMorin** [dclements/elixir_bbedit#3]
I'd like to be able to fold sections such as @doc, @moduledoc, and defp.

**Comment by dclements** [comment]
Good idea, I'll look into it when I update for Elixir 1.0.0 or someone else can tackle it if they feel up to it.

**Comment by RichMorin** [comment]
If you can point me in the right direction and get me started, I might be able to help.

**Comment by jolux** [comment]
It's been nearly a year since 1.0 was released. Any chance of this getting revved in the foreseeable future?

**Comment by chipotle** [comment]
In case anyone stumbles across this, it's probably worth pointing out that I have a new Elixir package that's updated for Elixir 1.14, with a bundled language server for BBEdit 14+. 

https://github.com/chipotle/Elixir.bbpackage

**Comment by beatrixada** [comment]
Awesome! Will take a look when I get the chance. Mostly moved on to VSCode and Emacs but I still miss aspects of BBEdit.
